catala/menjar/menjar.md

```diff
@@ -8,4 +8,5 @@ Et deixes anar i caus estès al terra.
 Què fas?
 
 No res, has mort. 
-I els metges no saben trobar una explicació, però de sucre no n'has menjat pas.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.